build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -110,13 +110,16 @@ steps:
       yarn gulp vscode-darwin-$(VSCODE_ARCH)-min-ci
     displayName: Build
     env:
+      GITHUB_TOKEN: $(github-distro-mixin-password)
```

Comment:
Is this for getting the internal electron build?

Reply:
Yes, as we'll be releasing internal builds to distro repo, using the same token to fetch release artifacts.